app/src/main/kotlin/com/mapzen/erasermap/view/ViewRecycler.kt

```diff
@@ -0,0 +1,35 @@
+package com.mapzen.erasermap.view
+
+import android.view.View
+import java.util.HashMap
+import java.util.HashSet
+
+class ViewRecycler {
```

Comment:
This seems a little spartan for a view recycler, at least compared to android's RecyclerView and iOS's myriad of recycling view components. This seems to be more like a singleton view holder so you never have more than one type of view, or am I reading it wrong?